docs/source/gbd2017_models/causes/neoplasms/breast_cancer_with_stage0/cancer_model.rst

```diff
@@ -0,0 +1,319 @@
+  |               |                      |       | metastatic_phase_of_breast_cancer (s_279)                    |
+  |               |                      |       | terminal_phase_of_breast_cancer (s_280)                      |
+  |               |                      |       | controlled_phase_of_breast_cancer_with_mastectomy (s_5486)   |
+  |               |                      |       | controlled_phase_of_breast_cancer_without_mastectomy (s_5489)| 
+  |               |                      |       | mastectomy_from_breast_cancer_beyond_ten_years (s_5492)      |
+  +---------------+----------------------+-------+--------------------------------------------------------------+
+
+.. image:: breast_cancer_hierarchy.svg
+
+
+Restrictions
+~~~~~~~~~~~~
+
+The following table describes any restrictions on the effects of this cause
+(such as being only fatal or only nonfatal), as well as restrictions on the age
+and sex of simulants to which different aspects of the cause model apply.
+
+.. list-table:: Restrictions
+   :widths: 15 15 20
+   :header-rows: 1
+
+   * - Restriction Type
+     - Value
+     - Notes
+   * - Male only
+     - False
+     -
+   * - Female only
+     - False
+     -
+   * - YLL only
+     - False
+     -
+   * - YLD only
+     - False
+     -
+   * - YLL age group start
+     - 15 to 19
+     - GBD age group id 8
+   * - YLL age group end
+     - 95 plus
+     - GBD age group id 235
+   * - YLD age group start
+     - 15 to 19
+     - GBD age group id 8
+   * - YLD age group end
+     - 95 plus
+     - GBD age group id 235
+
+
+Vivarium Modeling Strategy
+--------------------------
+
+Things to consider: 
+
+1. Within GBD 2017 data, there is no remission rate for stage 1+ breast cancer.
+2. Within GBD 2017, after diagnosis/treatment if a patient survives more than 10 years, they are considered cured for calculating disability. Additionally, per GBD 2017, the patients also do not have excess mortality rate after 10 years. For vivarium simulation models, this means we need to track how long patients have been in the breast cancer state. This should be easy for those who has a breast cancer incidence after sim start. For those who were iniatialized in the breast cancer state, we need to assign when breast cancer was diagnosised in the past. After their 10 years is up and they still havent died, we need to decide what susceptability state to remit them to. 
+
+.. todo::
+
+   Add more assumptions and limitations.
+
+
+Compartmental Diagram
++++++++++++++++++++++
+
+  .. image:: compartmental_model_simple.svg
+
+
+State and Transition Data Tables
+++++++++++++++++++++++++++++++++
+
++----------------------------------------------------------------------------+
+| State definitions                                                          |
++=======================+================+===================================+ 
+| State                 | State name     | Definition                        |
```

Comment:
Looks like State, State name, and Definition should be the names of the columns instead of cells within the columns, and "State definitions" should be the name of the table, not part of the table itself.

Reply:
Actually, looks like there is the same formatting issue with most of the tables...